web/concrete/src/Application/Application.php

```diff
@@ -130,6 +130,45 @@ public function clearCaches()
         OpCache::clear();
     }
 
+   /**
+     * Utility method for clearing page caches and block caches that has btCacheBlockOutputClearOnEvents option enabled
+     */
+    public function partialClearCaches()
+    {
+        \Events::dispatch('on_partial_cache_flush');
```

Comment:
Actually, you should just open a proposal for this. For this commit, it doesn't matter because you are within the application object right here, just do `$events = $this['director']`.